Library/Formula/python.rb

```diff
@@ -28,7 +29,7 @@ class Python < Formula
   depends_on "gdbm" => :recommended
   depends_on "openssl"
   depends_on "homebrew/dupes/tcl-tk" => :optional
-  depends_on "berkeley-db4" => :optional
+  depends_on "berkeley-db" => :optional
```

Comment:
I assume that we don't care that it is ```Deprecated since version 2.6: The bsddb module has been removed in Python 3.``` We still need to support this, correct?